crates/ruma-events-macros/src/event_parse.rs

```diff
@@ -226,13 +237,36 @@ impl Parse for EventEnumInput {
     }
 }
 
-pub struct EventEnumEntry {
-    pub attrs: Vec<Attribute>,
-    pub ev_type: LitStr,
+/// The entire `event_enum!` macro structure directly as it appears in the source code.
+pub struct EventEnumInputs {
+    pub(crate) enums: Vec<EventEnumInput>,
```

Comment:
I don't think there is a plural of `input`. How about

```suggestion
pub struct EventEnumInput {
    pub(crate) enums: Vec<EventEnumDecl>,
```

?

Reply:
Yeah I like that much better.